products/views.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import viewsets,status,mixins,generics 
+from products.models import Category, Brand, Product 
+from .serializers import CategorySerializer, BrandSerializer, ProductSerializer 
+from .pagination import ProductsResultSetPagination 
+
+
+class ProductViewSet(mixins.CreateModelMixin, 
+                    mixins.ListModelMixin, 
+                    mixins.RetrieveModelMixin, 
+                    mixins.DestroyModelMixin, 
+                    mixins.UpdateModelMixin, 
+                    viewsets.GenericViewSet): 
+    queryset = Product.objects.all() 
+    serializer_class = ProductSerializer 
+    pagination_class = ProductsResultSetPagination 
```

Comment:
So, any user has permissions to change data. Is it the way it should work?

More than, I'm sure the view wouldn't work appropriately on POST or PUT requests